trimmer_decorator.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'decorator'
+require_relative 'person'
+require_relative 'capitalize_decorator'
+
+class TrimmerDecorator < Decorator 
+
+  def initialize(nameable)
+    super
+    @nameable = nameable
+  end
+
+  def correct_name
+    super[1...10] 
+  end
```

Comment:
- [ ] Kindly fix this method so that it returns the correct string. Currently, this method returns the string missing the first character (See attached below). This is because you're ignoring the first index and starting from `1` in `super[1...10]`.
![image](https://user-images.githubusercontent.com/62290545/199438426-c552faa6-802d-461d-81e4-8fdd6e39e953.png)

Reply:
@ShadyShawkat you are right! Thank you for this!